pkg/apps/util/const.go

```diff
@@ -0,0 +1,49 @@
+package util
+
+const (
```

Comment:
@deads2k @tnozicka i tried to organize this a little... if the const is being used only by the utils itself, make it private... I will try to get rid of `DeploymentStatusAnnotation`.